test/functional/test_site/bugs/index.md

```diff
@@ -4,37 +4,4 @@ header: header.md
 </frontmatter>
 
 <div class="website-content">
-
```

Comment:
It seems like we now have an empty bugs file 🎉 

Should we leave a comment here so that future contributors know what this file is for, and where to place their bugs?

Reply:
It should probably be something like a template:
```
**Bug Description**
<a href=LINK_TO_MARKBIND_ISSUE>Issue #XX</a>

Repro:

Repro Steps...
```